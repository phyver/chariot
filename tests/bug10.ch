data nat where Zero : nat | Succ : nat -> nat

:type x (x 0)
