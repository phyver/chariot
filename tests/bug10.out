>> inductive type nat was succesfully defined
(x¹ (x² Zero³))
with types:
  - 1: nat → nat
  - 2: nat → nat
  - 3: nat
>> with free variables: x : nat → nat
